node/src/BaseClient.ts

```diff
@@ -7691,7 +7691,7 @@ export class BaseClient {
      *
      * @param password - The new password to update the current password, or `null` to remove the current password.
      * @param reAuth - If `true`, the client will re-authenticate immediately with the new password. If `false`, the new password will be used for the next connection attempt.
-     * @returns Always `"OK"`.
+     * @returns `"Ok"`, if `reAauth: true` returns `"OK"` if the reauthenticate succeed.
```

Comment:
What if reauth = false?

Reply:
@Yury-Fridlyand note: I removed completely the “will be used in the next” as I understand it creates confusion. It is a binary flag.